### PR TITLE
Package for Windows

### PR DESCRIPTION
**Pull request by Abbe98**
Supersedes #2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,4 @@
 build
-store
-dist
-nbproject\private
-/nbproject/private/
-bin
```
Comment (sebastian-berlin-wmse) [inline .gitignore:6]: This is still relevant. Even if it's removed by the clean target it may be left if you run other targets.

File: .gitignore [hunk 2]
```diff
@@ -1,7 +1,4 @@
...
+tmp
 pattypan.jar
+*.zip
```
Comment (sebastian-berlin-wmse) [inline .gitignore]: This could use a newline.

File: .gitignore (hunk 1 above)
Comment (sebastian-berlin-wmse) [reply .gitignore:6]: It's not quite clear in the UI, but this refers to bin/ (line 6).

File: .gitignore (hunk 2 above)
Comment (sebastian-berlin-wmse) [reply .gitignore]: ...at the end of the file.

Comment (Abbe98) [reply .gitignore]: indeed!

File: .gitignore (hunk 1 above)
Comment (Abbe98) [reply .gitignore:6]: Thanks!

**Pull request merged**